deploy/hooks/post-restore

```diff
@@ -8,4 +8,19 @@ if [ -x /etc/init.d/apache2 ]; then
     sudo /etc/init.d/apache2 restart
 fi
 
-exit 0
```

Comment:
2 concerns:
a) give varnish time to detect instance
b) stop deployment onto other instances when something is wrong with this instance.

This is simple aproach, on the safe side. Treating 2 concerns separately.